faq.md

```diff
@@ -256,11 +256,19 @@ or directly) dart:mirrors or dart:html.
 
 ### How big is the Flutter engine?
 
-In November 2015, we measured the size of a minimal Flutter app, bundled as
-an APK, to be approximately 8MB. For this simple app that used Material Design
-widgets, the core engine is approximately 5MB, the framework + app code is
-approximately 400kb, necessary Java code is 330k, and there is approximately
-2.5MB of ICU data.
+In June 2017, we measured the size of a minimal Flutter app
+(no Material Design, just a single `Center` widget, built with
+`flutter build apk`), bundled and
+compressed as a release APK, to be approximately 6.7MB.
+
+For this simple app,
+the core engine is approximately 3.3MB (compressed),
+the framework + app code is approximately 1.25MB bytes (compressed),
+necessary Java code (classes.dex) is 40k (compressed),
```

Comment:
The FLX is 56KB compresed here, the classes.dex is 42KB. It seems like if you mention one you should mention the other.